app/views/account.py

```diff
@@ -0,0 +1,630 @@
+            return response.success_response()
+        except NoResultFound:
+            logger.exception("Branch details does not exist")
+            response = ResponseGenerator(data={},
+                                         message="Branch details does not exist",
+                                         success=False,
+                                         status=HTTPStatus.NOT_FOUND)
+            return response.error_response()
+
+
+class BranchDetailsResourceId(Resource):
+    def get(self, branch_details_id):
+        """
+             This is GET API
+             Call this api passing a branch details id
+             parameters:
+                branch_code: String
+                branch_address: String
+             responses:
+                 404:
+                    description: Branch details with this id does not exist
+                 200:
+                     description: Branch details with this id return successfully
+                     schema:
+                         BranchDetailsSchema
+         """
+        try:
+            branch_details_data = BranchDetails.query.filter(BranchDetails.id == branch_details_id).first()
+            if not branch_details_data:
+                raise NoResultFound
+
+            result = branch_details_schema.dump(branch_details_data)
+
+            logger.info("Response for get with id request for branch details {}".format(result))
+            response = ResponseGenerator(data=result,
+                                         message="Branch details with this id return successfully",
+                                         success=True,
+                                         status=HTTPStatus.OK)
+            return response.success_response()
+        except NoResultFound:
+            logger.exception("Branch details id with this id does not exist")
+            response = ResponseGenerator(data={},
+                                         message="Branch details with this id does not exist",
+                                         success=False,
+                                         status=HTTPStatus.NOT_FOUND)
+            return response.error_response()
+
+    def put(self, branch_details_id):
+        """
+             This is PUT API
+             Call this api passing a branch details id
+             parameters:
+                branch_code: String
+                branch_address: String
+             responses:
+                 404:
+                    description: Branch details with this id does not exist
+                 200:
+                     description: Branch details with this id updated successfully
+                     schema:
+                         BranchDetailsSchema
+         """
+        try:
+            # retrieve body data from input JSON
+            data = request.get_json()
+            errors = branch_details_schema.validate(data, partial=True)
+            if errors:
```

Comment:
you are not printing `errors` anywhere here, please add it.

Reply:
Made the changes as suggested.